blockchain/hsm.go

```diff
@@ -55,17 +57,19 @@ func (a *BlockchainReactor) pseudohsmSignTemplates(ctx context.Context, x struct
 	Auth  string
 	Txs   []*txbuilder.Template `json:"transactions"`
 	XPubs []chainkd.XPub        `json:"xpubs"`
-}) []interface{} {
-	resp := make([]interface{}, 0, len(x.Txs))
+}) interface{} {
+	resp := make([]interface{}, len(x.Txs))
 	for _, tx := range x.Txs {
 		err := txbuilder.Sign(ctx, tx, x.XPubs, x.Auth, a.pseudohsmSignTemplate)
+		log.WithFields(log.Fields{"tx:": tx, "build err:": err}).Info()
```

Comment:
Don't append `:` after the field name. The log generated will look like `tx:=foo`, the normal style is `tx=foo`.

Reply:
Accept